includes/event/event-repository.php

```diff
@@ -533,3 +564,27 @@ function ( $host ) {
 		update_post_meta( $event->id(), '_hosts', $hosts_ids );
 	}
 }
+
+// phpcs:ignore Universal.Files.SeparateFunctionsFromOO.Mixed
+function add_user_id_where_clause_to_events_query( string $where, WP_Query $query ): string {
+	// Remove the filter, so it only applies to a single query.
+	remove_filter( 'posts_where', 'add_user_id_where_clause_to_events_query' );
```

Comment:
I haven't tested it but as far as I know it needs to have the exact same function parameter except for the arguments:
```suggestion
	remove_filter( 'posts_where', 'Wporg\TranslationEvents\Event\add_user_id_where_clause_to_events_query', 10 );
```

Reply:
This is a bug indeed, great catch. However, setting the correct value here makes tests fail, so I'm looking into what's happening.

Reply:
This is now fixed. Concerning the extra argument, `10`, it's not needed as `10` is the default value..